codecov.yml

```diff
@@ -1,26 +1,20 @@
 coverage:
   status:
-    patch:
-      default:
-        target: 0%
-        threshold: 0%
-      core-packages:
-        target: 0%
-        paths:
-          - packages
-      plugins:
-        target: 0%
-        paths:
-          - plugins
+    patch: off
     project:
       default:
         target: 0%
-        # send notification if coverage change exceed this threshold
-        threshold: 0%
-      core-packages:
+        threshold: 10%
+      core-packages-ts:
         target: 100%
         paths:
-          - packages
+          - packages/*.ts$
+          - packages/*.js$
+      core-packages-tsx:
+        target: 10%
+        paths:
+          - packages/*.jsx$
```

Comment:
would something like this work? 

```
- packages/!(superset-ui-control-utils)/**/*.{j,t}sx
```

https://www.digitalocean.com/community/tools/glob?comments=true&glob=packages%2F%21%28superset-ui-control-utils%29%2F%2A%2A%2F%2A.%7Bj%2Ct%7Dsx&matches=false&tests=%2F%2F%20This%20will%20match%20as%20it%20ends%20with%20%27.js%27&tests=packages%2Fsuperset-ui-control-utils%2Fworld.js&tests=packages%2Fsuperset-ui-control-utils%2Fworld.jsx&tests=packages%2Fsuperset-ui-control-utils%2Fworld.ts&tests=packages%2Fsuperset-ui-control-utils%2Fworld.tsx&tests=packages%2Fsuperset-ui-chart%2Fchart.tsx&tests=packages%2Fsuperset-ui-chart%2Fplugin%2Fchart.tsx&tests=%2F%2F%20This%20won%27t%20match%21&tests=%2Ftest%2Fsome%2Fglobs

Reply:
I thought codecovv glob works differently. Their documentation was not super clear.

Reply:
Yeah I'm always confused about the glob pattern. Let's give it a few iterations.

Reply:
Looks like `{j,t}` operator doesn't really work for codecov.

Reply:
does `@(j|t)` work?

Reply:
That looks even more advanced... I doubt codecov will support that.